### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by ro-savage** [rnosov/react-reveal#6]
Looks great @rnosov. Just some feedback

`Babel-runtime` is quite a large dependency. Why is it needed?

Could either the exact pollyfills needed be included or make the polyfills optional dependencies ?

`prop-types` could probably be a peerDependency as well.

You've included `stage-0` for babel. I had a very quick review of the source and couldn't see a reason to have `stage-0` (but easily could have missed it). Might be better to remove if its not needed?

**Comment by rnosov** [comment]
Thanks, @ro-savage 
You've got some good questions. 
`Babel-runtime` is needed because `transform-runtime` is being used. I've checked the compiled code of `react-reveal` and there are 6 `require` calls concerning `Babel-runtime`:
```javascript
require("babel-runtime/helpers/extends")
require("babel-runtime/helpers/objectWithoutProperties")
require("babel-runtime/helpers/classCallCheck")
require("babel-runtime/helpers/createClass")
require("babel-runtime/helpers/possibleConstructorReturn")
require("babel-runtime/helpers/inherits") 
```
My understanding of it is that only exact polyfills are included in the production code and not the `Babel runtime` in its entirety. They look like helpers for creation of ES6 classes so they would be needed for pretty much any modern React App and will be shared with other components. I don't think there is any point making them optional.

Regarding `prop-types` the [official advice](https://github.com/facebook/prop-types#how-to-depend-on-this-package) is to make it a dependency rather than a peerDependency. I see that there is a difference of opinions on that so perhaps you could elaborate why you would prefer it as a  peerDependency.

To be honest, `stage-0` is a leftover from a different project so I'll put a more appropriate stage in the next release although I don't think it'll affect production code in any way.

**Comment by rnosov** [comment]
Babel stage fixed in 0.5.1

**Comment by ro-savage** [comment]
Sounds great. Thanks for the feedback. Agree with everything you said. Nice job!
